examples/rln-js/index.js

```diff
@@ -150,34 +150,10 @@ async function initRLN(ui) {
   });
 
   ui.onRegister(async () => {
-    let event;
-
     ui.setRlnStatus("Trying to register...");
-    if (signature) {
-      event = await rlnContract.registerMember(rlnInstance, signature);
-    } else {
-      // if user imports credentials manually
-      // TODO(@weboko): add logic to cover it in RLNContract
-      const idCommitment = membershipKey.IDCommitment;
-      const reversedArray = idCommitment.slice().reverse();
-      const pubkey = ethers.utils.hexlify(reversedArray).toString();
-
-      const price = await rlnContract.MEMBERSHIP_DEPOSIT();
-
-      const signer = provider.getSigner();
-      const rlnContractWithSigner = rlnContract.connect(signer);
-
-      const txResponse = await rlnContractWithSigner.register(pubkey, {
-        value: price,
-      });
-      console.log("Transaction broadcasted:", txResponse);
-
-      const txReceipt = await txResponse.wait();
-
-      console.log("Transaction receipt", txReceipt);
-
-      event = txReceipt.events[0];
-    }
+    const event = signature
+      ? await rlnContract.registerMember(rlnInstance, signature)
+      : await rlnContract.registerMemberFromMembershipKey(membershipKey);
```

Comment:
Why do we have the switch? In what instance would we use the membership key and try to register?

Reply:
sort of unrelated, but would you prefer if we generated typechain assets?

Reply:
Since we allow to import manually I think we need that. 
https://github.com/waku-org/js-rln/pull/56#discussion_r1171722788

@rymnc, I am not sure I understand how typechain assets will help

Reply:
Would make it more type safe :)

Reply:
(if you choose to move this example to ts)

Reply:
Ok Sorry, Just remembered how the example work...

I think we may want to deprecate the "manual import" of RLN credentials for the example or split the examples into one with signature and one with manual import.

In the pure context of enabling a developer to use RLN in the browser, I would **only** give them credentials from signature.

the import function is only useful is:
- you created credentials in the past
- you created credentials with nwaku

Both scenarios above are more advanced and clutters the example.